java/java.lsp.server/vscode/schemas/ext.package.schema.json

```diff
@@ -0,0 +1,33 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "LSP Extension contributions to package.json",
+    "type": "object",
+    "properties": {
+        "contributes": {
+            "type": "object",
+            "properties": {
+                "nbls.documentSelectors" : {
```

Comment:
Q: the other (image resource map) uses `netbeans.` prefix. Shouldn't we use the same one ? If `nbls.` is now preferred (@jlahoda  ?), then please file an issue for me to fix the resource map setting.

Another question: is it recommended to add schema for each ferature ?

Reply:
Both features use `netbeans` prefix and are merged to the single schema now.